tests/pytorch/mock_sparse2/test_unary_op_sp.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import torch
+import sys
+
+import backend as F
+
+from dgl.mock_sparse2 import create_from_coo
+
+# FIXME(issue #4818): Skipping tests on win.
```

Comment:
`TODO(#4818):`

Reply:
updated in #4997
consider documenting it somewhere

Reply:
https://docs.google.com/document/d/1NGXPg4U67ZpMDVxxkb-SprEx11d8zqNB_0cLweTL90Y/edit#bookmark=id.rhioqnfrsekp

Reply:
Yes, I linked it to https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit#heading=h.4p68oec5ndrr, which is linked to PR template. Most important is we all be aware of this, and point it out in PR review. I don't expect every PR author read all the details when they code.